api/dps/server.go

```diff
@@ -17,6 +17,7 @@ package dps
 import (
 	"context"
 	"fmt"
+	"github.com/onflow/flow-dps/service/trace"
```

Comment:
potential linting issue

Reply:
+1. You should consider adding `golangci-lint` to the repo in a followup PR.

Reply:
I just added it to the CI flow in this PR :)